bin/oneoff/update_pd_applications_with_new_capitalization.rb

```diff
@@ -16,17 +16,14 @@
 
     principal_response = principal_application.sanitize_form_data_hash
 
-    response = principal_response.values_at(:replace_course, :replace_course_other).compact.join(": ")
-    replaced_courses = principal_response.values_at(:replace_which_course_csp, :replace_which_course_csd).compact.join(', ')
-    replace_course_string = "#{response}#{replaced_courses.present? ? ': ' + replaced_courses : ''}".gsub('::', ':')
+    replace_course_string = principal_response.values_at(:replace_course, :replace_course_other).compact.join(": ").gsub('::', ':')
```

Comment:
What is this oneoff script being updated to do? Usually, these are only run to update existing entries in the database, which hopefully, we don't need to do ... if we. *don't* need to run it, could you restore it?

Reply:
Oh yes will do! I just updated it just in case someone was going to run it